rootfs-builder/rootfs.sh

```diff
@@ -239,7 +239,13 @@ if [ -n "${USE_DOCKER}" ] ; then
 		--env KERNEL_MODULES_DIR="${KERNEL_MODULES_DIR}" \
 		--env EXTRA_PKGS="${EXTRA_PKGS}" \
 		--env OSBUILDER_VERSION="${OSBUILDER_VERSION}" \
+		--net host \
 		-v "${script_dir}":"/osbuilder" \
+		--cap-add NET_ADMIN \
+		--cap-add SYS_ADMIN \
+		--cap-add SYS_CHROOT \
+		--cap-add MKNOD \
+		 --security-opt apparmor:unconfined \
```

Comment:
Since these capabilities are only required for Ubuntu, I think the list of `docker run` options should be built up and these options only added when needed. Something like:

```
docker_run_args=""
docker_run_args+=" --rm"
docker_run_args+=" --runtime runc"

admin_caps=$(distro_needs_admin_caps "$distro")
if [ "$admin_caps" = "true" ]; then
    docker_run_args+=" --cap-add NET_ADMIN"
    docker_run_args+=" --cap-add SYS_ADMIN"
        :
fi

eval docker run "${docker_run_args}" ...
```